components/Media/src/Extensions/System.Threading.Tasks/AsyncMutex.cs

```diff
@@ -32,7 +32,7 @@ public async Task<IDisposable> LockAsync()
     /// <summary>
     /// Private class that implements the automatic release of the semaphore
     /// </summary>
-    private sealed class Lock : IDisposable
+    private sealed partial class Lock : IDisposable
```

Comment:
Remove `partial` here as well.

Reply:
Resolved in cedbe4db7eddca5cc8b87a4a4c8187af11cba38a